### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by amirhouieh**

### CODE REVIEW

File: research-areas.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Research areas
+---
+#overview
```
Comment (jm-clius) [inline research-areas.md:4]:
```suggestion
# Overview
```

File: vac.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Main
+---
+# Vac researches peer-to-peer, private, censorship resistant communication.
+
+
+## About Vac
+Vac is an R&D org creating modular p2p protocols for private, secure, censorship resistant communication.
+Researching All Layers
```
Comment (jm-clius) [inline vac.md:9]:
```suggestion
```

File: vac.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+
+### Generalized Messaging
+This includes both human to human communication, as well as machine to machine communication. Texting, data transfer, financial transactions, state channels and more.
+
+## Principles
+[missing content]
+
+## Research
+The research team is focused on research and specifications of protocols within the general scope of Vac. Waku v2 is the current main focus, and the main client is nim-waku.
```
Comment (jm-clius) [inline vac.md:31]:
```suggestion
The research team is focused on research and specifications of protocols within the general scope of Vac. Waku v2 is the current main focus, and the main client is [nwaku](https://github.com/status-im/nwaku) - the Nim implementation of Waku v2.
```

File: outreach/Platform outreach.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Platform outreach
```
Comment (oskarth) [inline outreach/Platform outreach.md:1]: This file can be removed

File: outreach/Vac research outreach.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Vac research outreach
```
Comment (oskarth) [inline outreach/Vac research outreach.md:1]: Ditto remove

File: outreach/Waku operator outreach.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# Waku operator outreach
```
Comment (oskarth) [inline outreach/Waku operator outreach.md:1]: Ditto remove

File: projects.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+title: Projects
+---
+# Vac projects
+There are several projects under Vac. These are largely independent and can be seen as specific bets.
+
+
+## Waku Research
+
+Basically what Vac is primarily focused on today. Serves Waku Product, and aims to making Waku a self-sustained robust network with strong and modular privacy guarantees. This also includes secure messaging research and things like p2p/networking.
+
```
Comment (oskarth) [inline projects.md:11]: As commented in hackmd, remove everything except first para here for now. We'll do greenfield content at a later stage. The hackmd this is based on was for private planning and not for public consumption at this stage

File: vac.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+### Modular System
+Provide options at each layer in the stack, instead of having a tightly coupled set of protocols. Allowing developers to choose what they use and ensure they know each choice comes with different trade-offs.
+
+
+### Generalized Messaging
+This includes both human to human communication, as well as machine to machine communication. Texting, data transfer, financial transactions, state channels and more.
+
+## Principles
+[missing content]
```
Comment (oskarth) [inline vac.md:28]: This can be copy paste from https://our.status.im/our-principles/ starting from

Comment (jm-clius) [reply vac.md:28]: @oskarth I have made these changes and will open for review soon.

File: vac.md [hunk 9]
```diff
@@ -25,13 +25,55 @@ Provide options at each layer in the stack, instead of having a tightly coupled
...
 [Go to research page](/research)
 
-Vac RFCs: https://rfc.vac.dev/
-Specs repo: https://github.com/vacp2p/rfc
-Board: https://github.com/orgs/vacp2p/projects/5
\ No newline at end of file
+Vac RFCs: https://rfc.vac.dev/ |
+Specs repo: https://github.com/vacp2p/rfc |
+Board: https://github.com/orgs/vacp2p/projects/5 |
+Forum: https://forum.vac.dev/
```
Comment (oskarth) [inline vac.md:79]:
```suggestion
- Vac RFCs: https://rfc.vac.dev/
- Specs repo: https://github.com/vacp2p/rfc
- Board: https://github.com/orgs/vacp2p/projects/5
- Forum: https://forum.vac.dev/
```

File: vac.md [hunk 10]
```diff
@@ -25,13 +25,55 @@ Provide options at each layer in the stack, instead of having a tightly coupled
...
+We create software incentivized to continue to exist and improve, without the stewardship of a single entity or any of the current team members.
+
+### X. Resourcefulness
+
+We are relentlessly resourceful. As we grow and have ready access to capital, it is our obligation to token holders to fight bureaucracy and inefficiencies within the organization. This means solving problems in the most effective way possible at lower economic costs (in terms of capital, time and resources).
 
 ## Research
 The research team is focused on research and specifications of protocols within the general scope of Vac. Waku v2 is the current main focus, and the main client is nim-waku.
 
 [Go to research page](/research)
```
Comment (oskarth) [inline vac.md:74]:
```suggestion
[Go to the research page](/research)
```

This page should show summary of posts probably, it is empty for me though

File: README.md [hunk 11]
```diff
@@ -1,43 +1 @@
 # Vac.dev Website
-
```
Comment (oskarth) [inline README.md:2]: Can we add instructions for how to run this locally?

File: research-log.md [hunk 12]
```diff
@@ -1,5 +0,0 @@
----
```
Comment (oskarth) [inline research-log.md:1]: Can we have some list of all the research log entries?

File: README.md (hunk 11 above)
Comment (amirhouieh) [reply README.md:2]: Sure, I will do this after the first launch :) 
Although for that you would need to clone and run a different repository! This repo is purely for content purposes.

File: research-log.md (hunk 12 above)
Comment (amirhouieh) [reply research-log.md:1]: Yes it is enabled 
https://staging.vac.dev/research

**Review by oskarth — APPROVED** [review]

**Review by jm-clius — APPROVED** [review]

**Pull request merged**